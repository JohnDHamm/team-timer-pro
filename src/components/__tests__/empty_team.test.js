import React from 'react'
import renderer from 'react-test-renderer'
import EmptyTeam from '../empty_team'

describe('<EmptyTeam />', () => {
  test('it renders correctly', () => {
    const tree = renderer.create(<EmptyTeam/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
});
