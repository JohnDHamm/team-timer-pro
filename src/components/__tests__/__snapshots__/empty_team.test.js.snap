// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyTeam /> it renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 85,
        "marginBottom": 10,
        "width": 85,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#A82F29",
        "fontFamily": "dosis-light",
        "fontSize": 25,
      }
    }
  >
    Your team is empty!
  </Text>
</View>
`;
